### PR TITLE
chore: Edited the query filter to remove .get()

### PR DESCRIPTION
**Pull request by shweta345**
Updated the code to resolve a customer feedback about understanding simple queries due to the .get() function.

### CODE REVIEW

**Comment by thatfiredev** [comment]
Hey @shweta345 
Can you please describe what issue the customer was running into? Links to public Github issues or internal bug tickets are welcome. Thanks!

**Comment by shweta345** [comment]
@thatfiredev Internal bug ID that it targets is http://b/300413440. I was unsure if I should link it here :)

**Comment by thatfiredev** [comment]
@shweta345 I do see the 2 simple queries that the customer seems to be talking about:

Screenshot 2023-11-28 at 11 22 29



But they don't seem to match with the query you're updating in this PR: 

https://github.com/shweta345/snippets-node/blob/c234a3db471da94e5c9be8a29d856e4a8df9e31e/firestore/main/index.js#L590

**Comment by shweta345** [comment]
@thatfiredev This is the link that customer has shared: https://cloud.google.com/firestore/docs/query-data/queries#simple_queries. If you see the second snippet, it is the one that I edited.

File: firestore/main/index.js [hunk 1]
```diff
@@ -587,7 +587,7 @@ async function queryAndFilter(db) {
   const citiesRef = db.collection('cities');
 
   // Create a query against the collection
-  const allCapitalsRes = await citiesRef.where('capital', '==', true).get();
+  const allCapitalsRes = await citiesRef.where('capital', '==', true);
```
Comment (thatfiredev) [inline firestore/main/index.js:590]: We can get rid of the `await` as well:
```suggestion
  const allCapitalsRes = citiesRef.where('capital', '==', true);
```

**Review by thatfiredev — CHANGES_REQUESTED** [review]
@shweta345 Ah, my bad. I was looking at the wrong snippet (modular SDK)

One minor change needed before we merge this:

**Comment by shweta345** [comment]
@thatfiredev Merged your edits. Thank you! :)

**Review by thatfiredev — APPROVED** [review]
LGTM
Thanks for updating this :)

**Pull request merged**